app/views/partials/_login_prompt.html.erb

```diff
@@ -7,10 +7,10 @@
     </div>
     <div class="card-body">
       <p class="card-text">
-        Log in with a C.S.V. Alpha account.
+        Log in with a <%= Rails.application.config.x.site_association %> account.
```

Comment:
'a' <association> and 'an' <association> would be a nice distinction linguistically but it's minor so this is fine